lesson11/js/main.js

```diff
@@ -0,0 +1,67 @@
+"use strict";
+const menuContainer = document.getElementById("js-menu_container");
+const menuLists = document.createElement("ul");
+const loading = document.getElementById("js-loading");
+
+function showLoadingImg() {
+  const loadingImg = document.createElement("img");
+  loadingImg.src = "images/loading-circle.gif";
+  loadingImg.classList.add("el_loadingImg");
+
+  loading.appendChild(loadingImg);
+}
+
+function createMenuList({data}) {
+  const fragment = document.createDocumentFragment();
+
+  data.forEach((value) => {
+      const listItem = document.createElement("li");
+  
+      const listImg = document.createElement("img");
+      listImg.src = value.img;
+      listImg.alt = value.alt;
+      listImg.width = 40;
+      listImg.height = 40;
+  
+      const listAnchor = document.createElement("a");
+      listAnchor.href = value.a;
+      listAnchor.textContent = value.text;
+  
+      fragment.appendChild(listItem).appendChild(listAnchor).appendChild(listImg);
+
+    });
+
+  menuContainer.appendChild(menuLists).appendChild(fragment);
+}
+
+async function fetchMenuContents(){
+  showLoadingImg();
+
+  const menuDataURL = 'https://jsondata.okiba.me/v1/json/Aticp210816214718';
+  const response = await fetch(menuDataURL);
+  
+  if(response.status === 200){
+    return await response.json();
```

Comment:
Since we are using `return response.json` here, we need to use `createMenuList({data})`.
If we take one more step, we can avoid the `split assignment`.
I'm sure it's easier than you think, so once you figure out how to do it, please try to figure it out!

Reply:
Thank you for review!

I understand that response.json () returns an javascript object, so I can use the usual array specification.
Thanks to that, I was able to make the code very simple.

[b5a3c2e](https://github.com/nyattsu72/hanson-frontend-cource/pull/14/commits/b5a3c2e55af358b61f64737b02fbd9492456b832)